Paymetheus/PurchaseTickets.xaml

```diff
@@ -0,0 +1,262 @@
+                may be chosen at any block for validation. The safest way to ensure that your vote 
+                succeeds is to use a stake pool.
+            </TextBlock>
+
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="10*"/>
+                    <ColumnDefinition Width="125"/>
+                    <ColumnDefinition Width="20"/>
+                    <ColumnDefinition Width="300"/>
+                    <ColumnDefinition Width="20*"/>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="25"/>
+                </Grid.RowDefinitions>
+                <TextBlock Grid.Column="1" Grid.Row="0" Style="{StaticResource shellDescriptionTextBlockStyle}" Text="Account" HorizontalAlignment="Right" VerticalAlignment="Center"/>
+                <ComboBox Tag="AccountComboBox" Grid.Column="3" Grid.Row="0" Height="20"
+                          ItemsSource="{Binding Source={StaticResource ViewModelLocator}, Path=SynchronizerViewModel.Accounts}"
+                          SelectedItem="{Binding SelectedAccount}"
+                          ItemTemplate="{StaticResource ComboBoxAccountDataTemplate}"
+                          Style="{DynamicResource ComboBoxStyleWithCurrency}" Foreground="#FF2970FF" ItemContainerStyle="{DynamicResource ComboBoxItemStyle}"/>
+
+                <TextBlock Grid.Column="1" Grid.Row="1" Style="{StaticResource shellDescriptionTextBlockStyle}" Text="Ticket address" HorizontalAlignment="Right" VerticalAlignment="Center"/>
+                <TextBox Grid.Column="3" Grid.Row="1" Name="TicketAddressTextBox" Height="20" VerticalAlignment="Center" TextAlignment="Left" 
+                         Text="{Binding TicketAddressString, Mode=TwoWay, ValidatesOnExceptions=True}" Style="{StaticResource tooltipAlertTextBox}"/>
+
+                <TextBlock Grid.Column="1" Grid.Row="2" Style="{StaticResource shellDescriptionTextBlockStyle}" Text="Using a pool?" HorizontalAlignment="Right" VerticalAlignment="Center"/>
+                <CheckBox Grid.Column="3" Grid.Row="2" Name="UsingAPoolCheckBox" Height="20" VerticalAlignment="Bottom" IsChecked="{Binding PoolChecked}"/>
+
+                <TextBlock Grid.Column="1" Grid.Row="3" Text="Pool address" HorizontalAlignment="Right" VerticalAlignment="Center">
+                    <TextBlock.Style>
+                        <Style TargetType="TextBlock">
+                            <Setter Property="Visibility" Value="Hidden" />
```

Comment:
all of these setters should be in the TextBlock itself, instead of set using setters in the Style